### DISCUSSION TITLE
datatable : Sort by and outpulLabel in header facet makes the sort not accessible (need to click outside the label)

### DISCUSSION THREAD

**Discussion by jebab** [primefaces/community#829] [PrimeFaces] [answered]
Tested with primetest 
Using Primefaces 11.0.0
Java 11 

The use case is : 
 - a datatable with a lot of small columns
 - Column header are abreviated
 - Need to add a title in case user wants to know to full name of the column

the following : 

    <h:form>
        <div class="card">
            <p:dataTable var="row" value="#{dtBasicView.products}">
                <p:column field="code" headerText="Code" sortable="false" filterable="false"/>
                <p:column field="name" headerText="Name"/>
                <p:column field="category" headerText="Category"/>
                <p:column field="quantity">
                    <f:facet name="header"><p:outputLabel title="Quantity">Quantity</p:outputLabel></f:facet>
                </p:column>
            </p:dataTable>
        </div>
    </h:form>

The sort wont work when clicking on the outputLabel, when clicking outside of it it's working.
How can i make the sort working when clicking on the ouputLabel ?

**Comment by melloware** [comment, marked as answer]
Don't use a `<p:outputLabel>` use a simple `<span>` like this it has the same effect except the sorting works on clicking it.

```xml
<f:facet name="header"><span title="Quantity">Quantity</span></f:facet>
```

**Reply by jebab** [reply]
Thanks !
